src/analyses/custom_bitvector_analysis.cpp

```diff
@@ -396,8 +395,11 @@ void custom_bitvector_domaint::transform(
         }
         else
         {
+          goto_programt::const_targett next = from;
+          ++next;
```

Comment:
That's a lacking `std::next` that I think goes in this PR.

Reply:
Fixed.